server/__tests__/api.test.js

```diff
@@ -77,3 +77,11 @@ describe('mixed-content-by-type', () => {
     expect(res.body.result).not.toBeNull();
   });
 });
+
+describe('key-exchange', () => {
+  it('Check /tls/key-exchange', async () => {
+    const res = await request(app).get('/api/tls/key-exchange');
+    expect(res.body.description).toBe('');
```

Comment:
Why is this test with an empty string? If we're not supposed to return anything, I think is okay to omit it, and instead, improve the test on body.result

Reply:
Improved this but removed some tests from this file for other visualizations, please tell me if this was a mistake on my part

Reply:
Seems okay. I merged everything into `feature/mixed-content-layouts` with little adaptations and tests are passing correctly